.github/workflows/ISSUE_TEMPLATE/enhancement.yaml

```
@@ -0,0 +1,19 @@
+name: Enhancement Tracking Issue
+description: Provide supporting details for a feature in development
+labels: [feature]
```

```suggestion
labels: [enhancement]
```

Let's use a label that already exists.